config/routes.rb

```diff
@@ -4,6 +4,7 @@
   resources :sessions, only: [:new, :create, :destroy]
   get "/signup", to: "users#new", as: :signup
   get "/login", to: "sessions#new", as: :login
+  get '/home', to: 'users#home', as: :home
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.